frontend/src/Components/BusinessDetailsSkeleton/BusinessDetailsSkeleton.css

```
@@ -0,0 +1,80 @@
+.business-details-skeleton {
+  padding: 0px 0 20px 0;
+}
+
+.skeleton-Header {
```

super nitpick, but I'd try to keep the naming convention consistent
```suggestion
.skeleton-header {
```

---

Alright